app/webpacker/styles/colours.scss

```diff
@@ -1,24 +1,25 @@
 $black: #1d1d1b;
-$blue: #2881be;
-$blue-dark: #0076bd;
+$blue: #00b1eb;
+$blue-dark: #1877cf;
```

Comment:
@garethwilliamsdesign This link blue that we were given is actually slightly worse in terms of colour contrast ratio (4.59 compared with our current 4.85).

Although this meets AA for normal, is it worth using the darker blue that was provided (#1557ac) instead which would then have a ratio of 7.02? But you may think this is not distinct enough from black - have a look and let's discuss!

Alternately we could consider using the GDS blue #1d70b8 (given we are using the GDS CTA green) which would have a ratio of 5.17 - this meets AA and is an improvement on what we have currently in production plus what we have been given as an updated colour

Reply:
I agree and am happy to use GDS blue. I'll make the change and we can review.